frontend/docs/publishing-packages.md

```diff
@@ -447,3 +447,39 @@ case, you can un-ignore the `dist/` directory by using a negation in the
 
 In this case, the `dist/` directory will be included when publishing, even
 though it is listed in the `.gitignore` file.
+
+## Publishing command line interfaces (CLIs)
+
+JSR supports publishing of command-line interfaces (CLIs) as part of a package.
+This allows package users to execute your CLI from the command line using
+`deno run`, `npx`, `yarn dlx`, and similar tools.
+
+Unlike npm, JSR does not have a special field in the `jsr.json` / `deno.json` to
+specify bin entrypoints. Instead, include the bin entrypoints in the `exports`
```

Comment:
That might make it difficult to publish existing packages to JSR, unless https://github.com/nodejs/modules/issues/274

Reply:
Why do you say so? Your bin entrypoint does not have to be in the `main`.

Reply:
I'm not exactly thrilled about having to resort to [hacks](https://github.com/nodejs/node/pull/32223#issuecomment-792909161), or rethink my `package.json` to match JSR.

Reply:
I'd be fine with this if JSR transpiled `import.meta.main` to something Node can understand.